mmv1/products/securityposture/posture.yaml

```diff
@@ -0,0 +1,519 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Posture'
+description: |
+  A Posture represents a collection of policy set including its name, state, description
+  and policy sets. A policy set includes set of policies along with their definition.
+  A posture can be created at the organization level.
+  Every update to a deployed posture creates a new posture revision with an updated revision_id.
+import_format: ['{{%parent}}/locations/{{location}}/postures/{{posture_id}}']
+base_url: '{{parent}}/locations/{{location}}/postures'
+
+self_link: '{{parent}}/locations/{{location}}/postures/{{posture_id}}'
+
+create_url: '{{parent}}/locations/{{location}}/postures?postureId={{posture_id}}'
+
+update_url: '{{parent}}/locations/{{location}}/postures/{{posture_id}}?revisionId={{revision_id}}'
+update_verb: :PATCH
+
+update_mask: true
+
+delete_url: '{{parent}}/locations/{{location}}/postures/{{posture_id}}'
+
+autogen_async: true
+# Sets parameters for handling operations returned by the API.
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: templates/terraform/constants/posture.go.erb
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'securityposture_posture_basic'
+    primary_resource_id: 'posture1'
+    test_env_vars:
+      org_id: :ORG_ID
+
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'parent'
+    description: |
+      The parent of the resource, an organization. Format should be `organizations/{organization_id}`.
+    immutable: true
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: "Location of the resource, eg: global."
+    immutable: true
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'postureId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Id of the posture. It is an immutable field.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      Name of the posture.
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      State of the posture.
+    required: true
+    values:
+      - :STATE_UNSPECIFIED
```

Comment:
```suggestion
```

TYPE_UNSPECIFIED values can generally be omitted as they're collapsed into `""` in the REST API